container/container_test.go

```diff
@@ -490,6 +490,26 @@ func TestNew(t *testing.T) {
 			},
 			wantContainerErr: true,
 		},
+		{
+			desc:     "fails when both SplitFixed and SplitPercent are specified",
```

Comment:
Sorry if commenting prematurely, not sure if you're still working on this.

Thank you for adding the test coverage for the new feature - can we also add a test case when the user specifies invalid SplitFixed value, like SplitFixed(-4) ?

The condition seems to be missing coverage:
https://coveralls.io/builds/23268685/source?filename=container%2Foptions.go#L263

Reply:
Not at all, you commented at the right point.

Will cover that condition. :+1: